contracts/Splitter.sol

```diff
@@ -11,6 +11,9 @@ contract Splitter is Pausable {
 
   mapping (address => uint256) public balances;
```

Comment:
You'll also need to update your migration script and test file to pass the value to the constructor when calling `new()`.